pybikes/data/cyclehire.json

```diff
@@ -0,0 +1,17 @@
+{
+    "instances": [
+        {
+            "tag": "cyclehire",
+            "meta": {
+                "latitude": 51.51135,
+                "longitude": -0.591562,
+                "city": "Slough",
+                "name": "Cycle Hire",
```

Comment:
Their website explicitly say it is called `Cycle Hire Slough`... but we agreed to take the name of the city out of the bike share system, to avoid redundant info like Cycle Hire Slough, Slough, UK. So :+1: